src/components/side-panel/layer-panel/layer-list.d.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Layer, LayerClassesType} from 'layers';
+import {Datasets} from 'reducers/vis-state-updaters';
+import * as VisStateActions from 'actions/vis-state-actions';
+import * as UIStateActions from 'actions/ui-state-actions';
+
+export type LayerListProps = {
+  datasets: Datasets;
+  layerClasses: LayerClassesType;
+  layers: Layer[];
+  layerOrder: number[];
+  uiStateActions: typeof UIStateActions;
+  visStateActions: typeof VisStateActions;
+  isSortable: Boolean;

```

Comment:
What should be the default value then?
I've put it to required since we will have only two displays (list or by-dataset) and in first we want it to be sortable and in other not to be sortable.

Reply:
I was thinking about if we should be explicit about what mode it should be in?